mimoto-issuers-config.json

```diff
@@ -408,6 +408,43 @@
       "credential_endpoint": "https://${mosip.esignet.insurance.host}/v1/esignet/vci/credential",
       "credential_audience": "https://${mosip.esignet.insurance.host}",
       "enabled": "true"
+    },
+    {
+      "credential_issuer": "VeridoniaCertify",
+      "display": [
+        {
+          "name": "Veridonia Insurance Company (Certify)",
+          "logo": {
+            "url": "https://sunbird.org/images/sunbird-logo-new.png",
```

Comment:
To be consistent, can we change this to https://${mosip.api.public.host}/inji/veridonia-logo.png

Reply:
changed this